SwissTransfer/Sources/SwissTransferApp.swift

```diff
@@ -23,6 +23,7 @@ import SwissTransferCore
 
 @main
 struct SwissTransferApp: App {
+    private let sentryService = SentryService()
```

Comment:
I did it this way, we can do it differently if you wish, lemme know.

Reply:
I think it's a good idea. It could also be the place where we put our debugging methods that we had in `SentryDebug` in other projects ?